src/web/scrapers.py

```diff
@@ -161,3 +168,8 @@ def _parse_external_id(self, url: str) -> str:
         parsed_url = urllib.parse.urlparse(url).path
         external_id = pathlib.PurePosixPath(urllib.parse.unquote(parsed_url)).parts[-1]
         return external_id
+
+    def _parse_tags(self, soup: BeautifulSoup) -> list[models.Tag]:
+        tags = soup.find_all("a", id=(lambda a: a and a.startswith("topics-link-")))
```

Comment:
I'm getting a type checking error on this line:
<img width="1231" alt="image" src="https://github.com/SpokaneTech/SpokaneTech_Py/assets/19392916/38f2a215-fa0c-4604-b19a-464c3be0c700">

Reply:
Changed this to a regex instead of a lambda.